src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -427,6 +428,22 @@
    [quo/button {:on-press #(theme/set-theme :light)} "Set light theme"]
    [quo/button {:on-press #(theme/set-theme :dark)} "Set dark theme"]])
 
+(defn category-view
+  []
+  (let [open? (reagent/atom false)]
+    (fn [category]
+      [rn/view {:style {:margin-vertical 8}}
+       [quo/dropdown {:selected @open? :on-change #(swap! open? not) :type :grey}
+        (clojure.core/name (key category))]
+       (when @open?
+         (for [{:keys [name]} (val category)]
```

Comment:
Thank you @flexsurfer for creating quo groups.

should we replace this with `component-name`? and remove `clojure.core/name`

Reply:
hm, idk, seems like not protected by lint? so we'll have this a lot still :(

Reply:
im confused :)